tests/virt_autotest/qem/live_migration_dst.pm

```diff
@@ -0,0 +1,49 @@
+# Copyright © 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Live migration test for destination host
+#
+# - Wait for guest migration initiated from source host after installation.
+#
+# Maintainer: Tony Yuan <tyuan@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use mmapi;
+use lib "sle/tests/virt_autotest";
+use virt_utils 'upload_virt_logs';
+
+sub run {
+    select_console 'root-ssh';
+
+    # create the lock
+    mutex_create 'dst_ready';
+
+    # wait until all children finish
+    wait_for_children;
+}
+
+sub post_fail_hook {
+    my $self = shift;
+
+    diag("live migration failed");
+    upload_virt_logs("/var/log/libvirt", "libvirt_log_dst");
```

Comment:
@tonyyuan1 Only libvirt log is not enough. Suggest to upload supportconfig log, which contains libvirt/qemu/xen/kernel/guest xml/etc logs. You can find existing functions in VT code.

Reply:
Openqa uploads a lot logs by default after a test fails. I am not sure if it includes  supportconfig. I will check. 
One missing piece here: set libvirtd to debug level before test.  This will be added soon.

Reply:
Will replace 'upload_virt_logs' with 'collect_virt_system_logs'.